webpack.dev.config.babel.js

```diff
@@ -59,14 +59,14 @@ export default {
     // messages, because they use
     // [Invariant](https://github.com/zertosh/invariant) which hides error
     // messages in the production build.
-    new webpack.NormalModuleReplacementPlugin(
-      /^react$/,
-      'react/umd/react.development.js',
-    ),
-    new webpack.NormalModuleReplacementPlugin(
-      /^react-dom$/,
-      'react-dom/umd/react-dom.development.js',
-    ),
+    // new webpack.NormalModuleReplacementPlugin(
+    //   /^react$/,
+    //   'react/umd/react.development.js',
+    // ),
+    // new webpack.NormalModuleReplacementPlugin(
+    //   /^react-dom$/,
+    //   'react-dom/umd/react-dom.development.js',
+    // ),
```

Comment:
Is this supposed to be commented out?  (should it be deleted instead?)

Reply:
Well... it's something I had to remove, and I was unable to find a replacement for it. I don't really want to just delete it, so I guess I should comment it out with a descriptive comment and open an issue to see if we can find a replacement, or determine if it's actually needed. I'll do that.